lapce-ui/src/editor/tab_header_content.rs

```diff
@@ -362,6 +458,18 @@ impl Widget<LapceTabData> for LapceEditorTabHeaderContent {
                         if let Some(file_name) = path.file_name() {
                             if let Some(s) = file_name.to_str() {
                                 text = s.to_string();
+                                let nb = name_count
+                                    .get(file_name.to_str().unwrap())
+                                    .unwrap();
+                                if *nb > 1 {
+                                    file_path = format!(
+                                        " {}",
+                                        self.get_effective_path(
+                                            data.workspace.path.clone(),
+                                            path
+                                        )
```

Comment:
`get_effective_path` could take an `Option<&PathBuf>` to avoid unneeded cloning. (`data.workspace.path.as_ref()` to get a reference into the option)